app/src/main/java/ca/deflector/passable_shots/MainActivity.java

```diff
@@ -44,7 +45,20 @@ public void onPictureTaken(byte[] data, Camera camera) {
                 // call encryption here
                 try {
                     byte[] encrypted = EncryptionSystem.encrypt(data);
-                    FileOutputStream fos = new FileOutputStream(Environment.getExternalStorageDirectory() + "/passable." + System.currentTimeMillis());
+                    // determine latest number
+                    int maxPassableFile = 0;
+                    for (File f : Environment.getExternalStorageDirectory().listFiles())
```

Comment:
I disagree. It was a comment about (a) code readability and (b) the potential for future bugs, which both go beyond coding standards. I think it's quite appropriate for CR.